### PR TITLE
Widgets ranking documentation

### PR DESCRIPTION
**Pull request by th1988**

### CODE REVIEW

File: source/includes/_widgets.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "data": [
+    {
+      "id": "ranking",
+      "type": "widgets",
+      "attributes": {
+        "company_wide_ranking": "23",
```
Comment (SweeD) [inline source/includes/_widgets.md]: I would like to see the rank as an integer. What do you think?

**Review by SweeD — COMMENTED** [review]
Just a small question. Rest looks good to me.  👍

File: source/includes/_widgets.md (hunk 1 above)
Comment (th1988) [reply source/includes/_widgets.md]: Good point. I'll change it.

Comment (th1988) [inline source/includes/_widgets.md]:
```suggestion
        "company_wide_ranking": 23,
```

File: source/includes/_widgets.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+{
+  "data": [
+    {
+      "id": "ranking",
+      "type": "widgets",
+      "attributes": {
+        "company_wide_ranking": 23,
+        "company_employee_amount": "120",
+        "department_wide_ranking": "12",
```
Comment (th1988) [inline source/includes/_widgets.md]:
```suggestion
        "department_wide_ranking": 12,
```

File: source/includes/_widgets.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Content-Type: application/json
+
+{
+  "data": [
+    {
+      "id": "ranking",
+      "type": "widgets",
+      "attributes": {
+        "company_wide_ranking": 23,
+        "company_employee_amount": "120",
```
Comment (th1988) [inline source/includes/_widgets.md]:
```suggestion
        "company_employee_amount": 120,
```

File: source/includes/_widgets.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        "department_wide_ranking": "12",
+        "department_employee_amount": "30"
```
Comment (th1988) [inline source/includes/_widgets.md]:
```suggestion
        "department_employee_amount": 30
```

**Comment by th1988** [comment]
@SweeD changes done 😄

**Comment by SweeD** [comment]
Looks good to me. 👏 

Please go ahead and 🚢 at your will. 👍

File: source/includes/_widgets.md [hunk 5]
```diff
@@ -77,3 +77,46 @@ type               | Type of returned widgets
...
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "data": [
+    {
+      "id": "ranking",
```
Comment (th1988) [inline source/includes/_widgets.md]: The id should be `rankings` instead of `ranking` what do you think @SweeD ?

Comment (th1988) [inline source/includes/_widgets.md]:
```suggestion
      "id": "rankings",
```

**Pull request merged**